test/nsql_bench/README.md

```diff
@@ -14,17 +14,104 @@ curl -XPOST "http://localhost:8090/v1/evals/tpch_nsql" \
 [{"id":"d846dcc0a0dd9df94b8d4f72c874aa90","created_at":"2025-02-04T23:10:29","dataset":"tpch_nsql","model":"gpt-4o-mini","status":"Completed","scorers":["json_match"],"metrics":{"json_match/mean":0.6363636}}]
 ```
 
-To review benchmark detailed result:
+To review model answers and score details:
 
 ```bash
-curl -XPOST http://localhost:8090/v1/sql \
-  --data "SELECT
-    input,
-    output,
-    actual,
-    value
-  FROM eval.results
-  WHERE run_id='74d3c352a21966af0f1a5a56fc85f4b8'" | jq
+curl -XPOST http://localhost:8090/v1/sql --data "
+WITH latest_run AS (
+  SELECT id FROM spice.eval.runs ORDER BY created_at DESC LIMIT 1
+)
+SELECT run_id, input, output, actual, value
+FROM eval.results
+WHERE run_id = (SELECT id FROM latest_run)
+" | jq
+```
+
+## Performance metrics
+
+### Main Metrics
+
+```sql
+WITH latest_run AS (
+    SELECT id, created_at, EXTRACT(EPOCH FROM (completed_at - created_at)) AS duration_seconds
+    FROM spice.eval.runs
+    ORDER BY created_at DESC LIMIT 1
+),
+mean_score AS (
```

Comment:
No averages!

Reply:
Yeah, bad naming, this is more of an `overall score` based on individuals tests scores (0 or 1). I'll update this as part of next PR. Thank you!